lib/styles/geshi.less

```diff
@@ -124,4 +124,13 @@
     .re1, .st0, .st_h {
         color: #ff0000;
     }
-}
\ No newline at end of file
+
+    li, .li1 {
+        font-weight: normal;
+        vertical-align:top;
+    }
+
+    .ln-xtra {
+        background-color: #ffc;
```

Comment:
Might be. That is the color that geshi is using for highlighting. The colors are nearly the same compared to the standard dokuwiki template. Maybe we should not use @ini_highlight here because I assume that in other templates with a very different color for @ini_highlight this might not fit good to the other geshi colors.